pkg/minikube/bootstrapper/bsutil/kverify/node_health.go

```diff
@@ -0,0 +1,145 @@
+				out.WarningT("The node on {{.name}} has ran out of disk space. please consider allocating more disk using or pruning un-used images", out.V{"name": n.Name})
+				if driver.IsKIC(drver) && runtime.GOOS != "linux" {
+					out.T(out.Stopped, "Please increase Docker Desktop's disk image size.")
+					if runtime.GOOS == "darwin" {
+						out.T(out.Documentation, "Documentation: {{.url}}", out.V{"url": "https://docs.docker.com/docker-for-mac/space/"})
+					}
+					if runtime.GOOS == "windows" {
+						out.T(out.Documentation, "Documentation: {{.url}}", out.V{"url": "https://docs.docker.com/docker-for-windows/"})
+					}
+				} else { // VM-drivers
+					out.T(out.Stopped, "You can specify a larger disk for your cluster using `minikube start --disk` ")
+				}
+				out.Ln("") // if there is error message, lets make an empty space for better visilibtly
+				return fmt.Errorf("node %q has unwanted condition %q : Reason %q Message: %q ", n.Name, c.Type, c.Reason, c.Message)
+			}
+
+			if c.Type == v1.NodeMemoryPressure && c.Status == v1.ConditionTrue {
+				out.Ln("")
+				out.ErrT(out.FailureType, "node {{.name}} has unwanted condition {{.condition_type}} : Reason {{.reason}} Message: {{.message}}", out.V{"name": n.Name, "condition_type": c.Type, "reason": c.Reason, "message": c.Message})
+				out.WarningT("The node on {{.name}} has ran of memory.", out.V{"name": n.Name})
+				if driver.IsKIC(drver) && runtime.GOOS != "linux" {
+					out.T(out.Stopped, "Please increase Docker Desktop's memory.")
+					if runtime.GOOS == "darwin" {
+						out.T(out.Documentation, "Documentation: {{.url}}", out.V{"url": "https://docs.docker.com/docker-for-mac/space/"})
+					}
+					if runtime.GOOS == "windows" {
+						out.T(out.Documentation, "Documentation: {{.url}}", out.V{"url": "https://docs.docker.com/docker-for-windows/"})
+					}
+				} else {
+					out.T(out.Stopped, "You can specify a larger memory size for your cluster using `minikube start --memory` ")
+				}
+				out.Ln("") // if there is error message, lets make an empty space for better visilibtly
+				return fmt.Errorf("node %q has unwanted condition %q : Reason %q Message: %q ", n.Name, c.Type, c.Reason, c.Message)
+			}
+
+			if c.Type == v1.NodePIDPressure && c.Status == v1.ConditionTrue {
+				out.Ln("")
+				out.ErrT(out.FailureType, "node {{.name}} has unwanted condition {{.condition_type}} : Reason {{.reason}} Message: {{.message}}", out.V{"name": n.Name, "condition_type": c.Type, "reason": c.Reason, "message": c.Message})
+				out.WarningT("The node has ran out of available PIDs.", out.V{"name": n.Name})
+				out.Ln("")
+				return fmt.Errorf("node %q has unwanted condition %q : Reason %q Message: %q ", n.Name, c.Type, c.Reason, c.Message)
+			}
+
+			if c.Type == v1.NodeNetworkUnavailable && c.Status == v1.ConditionTrue {
+				out.Ln("")
```

Comment:
Send the space to stderr with the other lines.

Reply:
how ?